ProcessMaker/processmaker
# Integrate Ellipses Menu pt1

## Issue & Reproduction Steps

Ticket [FOUR-8029](https://processmaker.atlassian.net/browse/FOUR-8029)

## Solution
Implements the new ellipsis menu in the following areas:

- Tasks/To Do Tasks
- Designer/Processes
- Designer/Processes (categories)
- Designer/Processes (archived)
- Designer/Processes (templates)
- Designer/Scripts
- Designer/Scripts (categories)
- Designer/Screens
- Designer/Screens (categories)
- Designer/Environment Variables
- Admin/Users
- Admin/Groups
- Admin/Auth Clients
- Admin/Customize UI/Dashboards
- Admin/Customize UI/Menus


## How to Test

Ensure you have the Dyanmic UI Package installed using branch[ `task/FOUR-8029`](https://github.com/ProcessMaker/package-dynamic-ui/pull/80)
1. Check that all areas listed above have the new Ellipsis menu properly configured based on the actions available for each one. This means that the menu should display all relevant options and functions, and that any unnecessary items are removed from the dropdown menu.

2. Ensure that permissions are correctly supported. Users with appropriate permissions should be able to access and use the menu, while users without those permissions should not be able to see or interact with it.

3. Test that selecting actions from the menu that do not open a modal or run a function is a proper link. These links should be able to be opened in a new tab by selecting the "command" or "ctrl" key on your keyboard while clicking on the link.

## Related Tickets & Packages
-[ Dynamic UI Package PR ](https://github.com/ProcessMaker/package-dynamic-ui/pull/80)

## Code Review Checklist


[FOUR-8029]: https://processmaker.atlassian.net/browse/FOUR-8029?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
